pkg/addon/create.go

```diff
@@ -242,6 +242,7 @@ func createSampleFiles(addonName, addonPath string) error {
 
 // writeRequiredFiles creates required files for an addon,
 // including template.yaml, readme.md, and metadata.yaml
+//nolint:gosec
```

Comment:
// nolint:gosec

Reply:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.

From [here](https://golangci-lint.run/usage/false-positives/#Use%20//nolint%20instead%20of%20//%20nolint%20because%20machine-readable%20comments%20should%20have%20no%20space%20by%20Go%20convention.:~:text=tests%20for%20it.-,Use,because%20machine%2Dreadable%20comments%20should%20have%20no%20space%20by%20Go%20convention.,-Edit%20this%20page).

That was why I wrote it without the space.

Reply:
It's reasonable. You are right. We can make it without the extra space.